src/index.ts

```diff
@@ -98,7 +102,14 @@ export function throttling(octokit: Octokit, octokitOptions = {}) {
   const events = {};
   const emitter = new Bottleneck.Events(events);
   // @ts-ignore
-  events.on("abuse-limit", state.onAbuseLimit);
+  events.on(
+    "secondary-limit",
+    state.onSecondaryRateLimit ||
+      deprecate(
+        state.onAbuseLimit,
+        "'onAbuseLimit' is deprecated, use 'onSecondaryRateLimit' instead"
+      )
```

Comment:
@wolfy1339 
This is the method you expected to be deprecated right?

Is `util/deprecate` good in this case??

Reply:
> This is the method you expected to be deprecated right?

Yes

> Is `util/deprecate` good in this case??

That's a NodeJS only function. Octokit can be run in a browser environment.
Take example from this commit:
https://github.com/octokit/webhooks.js/commit/f8f3d15c9fe10772010420dae0c77b0920cde088

Reply:
Change applied.
For the `@deprecated` annotation I think it has sense to improve types for `octokitOptions` so we can mark it as deprecated. What do you think @wolfy1339 ?